.codecov.yml

```diff
@@ -18,12 +18,12 @@ coverage:
   range: "30...90"
   precision: 2
   round: down
-  status:
+  state:
```

Comment:
Think this highlights some issues around global search and replace? Before I review this can you please just double check that all `status` -> `state` replacements in this PR are valid (this one is not!)?